Mathlib/Algebra/Order/Field/Basic.lean

```diff
@@ -435,6 +435,16 @@ theorem inv_pow_anti (a1 : 1 ≤ a) : Antitone fun n : ℕ => (a ^ n)⁻¹ := fu
 theorem inv_pow_strictAnti (a1 : 1 < a) : StrictAnti fun n : ℕ => (a ^ n)⁻¹ := fun _ _ =>
   inv_pow_lt_inv_pow_of_lt a1
 
+theorem le_iff_forall_one_lt_le_mul₀ {α : Type*} [LinearOrderedField α]
```

Comment:
```suggestion
theorem le_iff_forall_one_lt_le_mul₀ {α : Type*} [LinearOrderedSemifield α]
```
This was suggested on Zulip to also work.

Other than that, LGTM. (I edited your commit message btw)

bors d+